### PR TITLE
Update wording, screenshots, URL and icon

### PR DESCRIPTION
**Pull request by svelle**
This PR updates all the wording and graphics to be more in line with our current product offering. 
Also mattermost.org will be phased out soon so that got fixed as well :)

### CODE REVIEW

File: com.mattermost.Desktop.appdata.xml [hunk 1]
```diff
@@ -5,27 +5,30 @@
...
+	flexible, and deeply integrated with the tools you love. 
+	An alternative to SaaS collaboration solutions, 
+	Mattermost is built explicitly for developer teams, tools, and workflows. 
+	It is written in Golang and React and runs as a production-ready Linux binary under an MIT license with either MySQL or Postgres.
     </p>
   </description>
   <screenshots>
     <screenshot type="default">
-      <image type="source">https://i.ibb.co/r0pgQMC/111417-mattermost-1513705902-5351114.png</image>
+      <image type="source">http://mattermost.com/wp-content/uploads/2022/01/FlatHub_Linux_Channels_Hero.jpg</image>
```
Comment (vchernin) [inline com.mattermost.Desktop.appdata.xml:22]:
```suggestion
      <image type="source">https://mattermost.com/wp-content/uploads/2022/01/FlatHub_Linux_Channels_Hero.jpg</image>
```

File: com.mattermost.Desktop.appdata.xml [hunk 2]
```diff
@@ -5,27 +5,30 @@
...
     </screenshot>
     <screenshot>
-      <image type="source">https://i.ibb.co/bsLJ7D6/com-mattermost-Desktop-52afcc85d8efb3f60a3e99ee297042e2.png</image>
+      <image type="source">http://mattermost.com/wp-content/uploads/2022/01/FlatHub_Linux_Playbooks_Hero.jpg</image>
```
Comment (vchernin) [inline com.mattermost.Desktop.appdata.xml:25]:
```suggestion
      <image type="source">https://mattermost.com/wp-content/uploads/2022/01/FlatHub_Linux_Playbooks_Hero.jpg</image>
```

File: com.mattermost.Desktop.appdata.xml [hunk 3]
```diff
@@ -5,27 +5,30 @@
...
     </screenshot>
     <screenshot>
-      <image type="source">https://i.ibb.co/x71fXYQ/com-mattermost-Desktop-a4b0e543c523f07a83028d9c2dc9bd6a.png</image>
+      <image type="source">http://mattermost.com/wp-content/uploads/2022/01/FlatHub_Linux_Boards_Hero.jpg</image>
```
Comment (vchernin) [inline com.mattermost.Desktop.appdata.xml:28]:
```suggestion
      <image type="source">https://mattermost.com/wp-content/uploads/2022/01/FlatHub_Linux_Boards_Hero.jpg</image>
```

File: com.mattermost.Desktop.appdata.xml [hunk 4]
```diff
@@ -5,27 +5,30 @@
...
+    </screenshot>
+    <screenshot>
+      <image type="source">http://mattermost.com/wp-content/uploads/2022/01/FlatHub_Linux_Integrations_Hero.jpg</image>
```
Comment (vchernin) [inline com.mattermost.Desktop.appdata.xml:31]:
```suggestion
      <image type="source">https://mattermost.com/wp-content/uploads/2022/01/FlatHub_Linux_Integrations_Hero.jpg</image>
```

**Review by vchernin — APPROVED** [review]
looks good, but probably best to use https (that site redirects http->https anyhow)

**Comment by svelle** [comment]
> looks good, but probably best to use https (that site redirects http->https anyhow)

oops, you're totally right, missed that while copy pasting. thanks!

**Pull request merged**